bpf/bpf_netdev.c

```diff
@@ -270,7 +271,7 @@ resolve_srcid_ipv4(struct __ctx_buff *ctx, const struct iphdr *ip4,
 
 	/* Packets from the proxy will already have a real identity. */
 	if (identity_is_reserved(srcid_from_ipcache)) {
-		info = ipcache_lookup4(&IPCACHE_MAP, ip4->saddr, V4_CACHE_KEY_LEN);
+		info = lookup_ip4_remote_endpoint(ip4->saddr);
```

Comment:
Don't we need the same fix for the IPv6 path (i.e., `resolve_srcid_ipv6`)?
/cc @joestringer

Reply:
I think yes - but the implementation of `ipcache_lookup6` is slightly different and I've only run tests with ipv4 packets

Reply:
I think it wouldn't hurt to make the equivalent change there, as long as it passes CI.

We can separately review how well we cover IPv6 in each of the CI tests but I suspect that the scope of that change is bigger than what we intend to address in this PR.

Reply:
added the ipv6 change as well, would be good to run CI again

Reply:
@willdeuschle I'm not seeing it. Maybe you forgot to push?

Reply:
apologies - wrong remote. it's fixed now